### PR TITLE
Added metrics endpoint - Closes #137

### LINKED ISSUE #137: Please support Prometheus metrics
It would be nice to add `/metrics` endpoint with Prometheus metrics, following [Reporting Metrics To Prometheus In ASP.NET Core](https://www.c-sharpcorner.com/article/reporting-metrics-to-prometheus-in-asp-net-core/) is straightforward.

### PR DESCRIPTION
**Pull request by cesinhaugusto**
Added a new endpoint with Prometheus metrics using [prometheus-net](https://github.com/prometheus-net/prometheus-net) library.

A HTTP request middleware was created and placed into a new folder called Metrics. Required injections were added to Startup.cs

### CODE REVIEW

File: src/WebApi/DependencyInjection/Metrics/RequestMiddleware.cs [hunk 1]
```diff
@@ -0,0 +1,52 @@
+namespace WebApi.DependencyInjection.Metrics
```
Comment (ivanpaulovich) [inline src/WebApi/DependencyInjection/Metrics/RequestMiddleware.cs]: Please remove. There is a default middleware that generates a histogram with the data https://github.com/prometheus-net/prometheus-net/blob/master/Prometheus.AspNetCore/HttpMetricsMiddlewareExtensions.cs

File: src/WebApi/Startup.cs [hunk 2]
```diff
@@ -43,6 +45,8 @@ public void ConfigureDevelopmentServices(IServiceCollection services)
             IApiVersionDescriptionProvider provider)
         {
             app.UseHttpsRedirection();
+            app.UseMetricServer();
```
[the remark this answers is not in the document]
Comment (cesinhaugusto) [reply src/WebApi/Startup.cs]: Did it, and created the HttpMetricsExtensions with the counter options in order to inject into the app configuration.

File: src/WebApi/DependencyInjection/HttpMetricsExtensions.cs [hunk 3]
```diff
@@ -0,0 +1,24 @@
+namespace WebApi.DependencyInjection
+{
+    using Microsoft.AspNetCore.Builder;
+    using Prometheus;
+
+    public static class HttpMetricsExtensions
+    {
+        public static IApplicationBuilder UseMangaHttpMetrics(this IApplicationBuilder appBuilder)
+        {
+            return appBuilder.UseHttpMetrics(options =>
```
Comment (ivanpaulovich) [inline src/WebApi/DependencyInjection/HttpMetricsExtensions.cs]: Super 😃

**Comment by cesinhaugusto** [comment]
Reviewed the code in order to use prometheus-net built-in HTTP request middleware, instead of creating a personalized one.

**Comment by ivanpaulovich** [comment]
@cesinhaugusto may I add your name to the contributors list in the readme page?

**Comment by cesinhaugusto** [comment]
Of course @ivanpaulovich! I'm excited to contribute 😃

**Comment by ivanpaulovich** [comment]
@all-contributors please add @cesinhaugusto for code.

**Pull request merged**